app/apis/v1/logout_api.rb

```diff
@@ -0,0 +1,34 @@
+module V1
```

Comment:
I think it would be necessary only when the login records are recorded.
Or, it does not mean anything when user click logout function.
Could you confirm it? Please think about the user flow.

Reply:
I put the logout function here first as a frame for further design. Like you said, it doesn't really mean anything for now.
However, we will discuss the significance of the logout function in this project, then complete / remove the function base on our final decision.